forms/HtmlEditorField.php

```diff
@@ -674,7 +680,9 @@ protected function getFieldsForOembed($url, $file) {
 					'center' => _t('HtmlEditorField.CSSCLASSCENTER', 'Centered, on its own.'),
 					'left' => _t('HtmlEditorField.CSSCLASSLEFT', 'On the left, with text wrapping around.'),
 					'right' => _t('HtmlEditorField.CSSCLASSRIGHT', 'On the right, with text wrapping around.')
-				)
+				),
+				Config::inst()->get('HtmlEditorField', 'media_alignment')
```

Comment:
I would put this config on the HtmlEditorField_Toolbar, since it is the sole consumer of this option.

Then you can use `static::config()->get('media_alignment')` and we won't have to manually rewrite the class name on merge to 4. :)